rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.split/40_routing_partition_size.yml

```diff
@@ -5,9 +5,9 @@ more than 1:
         wait_for_active_shards: 1
         body:
           settings:
-            index.number_of_shards: 2
+            index.number_of_shards: 3
             index.number_of_replicas: 0
-            index.number_of_routing_shards: 4
+            index.number_of_routing_shards: 6
             index.routing_partition_size: 2
```

Comment:
`routing_partition_size` was == `number_of_shards` which is not allowed?

From https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html

> The number of shards a custom [routing](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-routing-field.html) value can go to. Defaults to 1 and can only be set at index creation time. This value must be less than the index.number_of_shards unless the index.number_of_shards value is also 1.